spec/features/super_admin/authorised_email_domain_spec.rb

```diff
@@ -37,6 +37,21 @@
         end
       end
 
+      # context 'given I want to delete a whitelisted domain' do
+      #   it 'removes a domain' do
+      #     create(:authorised_email_domain, name: 'police.uk')
+      #     visit admin_authorised_email_domains_path
+
+      #     expect { click_on 'Delete' }.to change { AuthorisedEmailDomain.count }.by(-1)
+      #     expect(page).to have_content("#{authorised_email_domain.name} has been deleted")
```

Comment:
The reason you are gettin...
```
undefined local variable or method 'authorised_email_domain' for #<RSpec::ExampleGroups::DELETEAuthorisedEmailDomainsId::
```

Is because `authorised_email_domain` is not defined. 

At L42 you create an authorised email domain using a factory `create(:authorised_email_domain, name: 'police.uk')` but you do not save this object to a variable. If you want to save it to a variable so you can refer to it later you should do something like:
`let(:authorised_email_domain,) { create(:authorised_email_domain, name: 'police.uk') }`